exla/mix.exs

```diff
@@ -143,6 +143,9 @@ defmodule EXLA.MixProject do
         "0" ->
           :none
 
+        "false" ->
```

Comment:
I would do this:

```elixir
case System.get_env("EXLA_FORCE_REBUILD") do
  value when value in [nil, "0", "false"] ->
    :none

  "partial" ->
    :partial

  value when value in ["true", "1"] ->
    :full

  other ->
    ...
end
```

A cond also works.

Reply:
I think the readability is either equal or worse in that alternative, given it's a small set of possible values

Reply:
With `cond` readability should be just as good as today and it should be easier to scan through. Your call :)

Reply:
Agreed this would suit cond perfectly.